README.md

```diff
@@ -72,7 +90,7 @@ If you opted to keep them in place, the server binaries are located in the bin f
     $ ./bin/CharServer&
     $ ./bin/MapServer&
 
-After running for the first time, the servers will create a config file in the current directory.
+After running for the first time, the servers will create a config file in the current directory that you need to edit.
```

Comment:
We probably want to explain what each option does and how you can modify it. A help page in the github wiki?